### PR TITLE
Code/js/loop

### PR DESCRIPTION
**Pull request by ntohidi**
Hi I really did it this time, task 1 and task 2 ready

### CODE REVIEW

**Comment by unclecode** [comment]
Again you forgot task 3, r u ok?

**Comment by ntohidi** [comment]
ok I did rask 3 plz check

File: task3/task.py [hunk 1]
```diff
@@ -1,3 +1,6 @@
 #Write a function to print even numbers from 1 to 10
 def printEven():
+    for x in range(1, 10):
```
Comment (unclecode) [inline task3/task.py]: good remember it should b 11 not 10

**Review by unclecode — CHANGES_REQUESTED** [review]
check my comments for task 3

**Comment by unclecode** [comment]
perfect change your readme file

**Pull request merged**